test/integration/orders.test.js

```diff
@@ -19,16 +19,13 @@ describe('Orders Integration', function () {
 
   it('supports create orders with a total price', async function () {
     const createOrderResponse = await patch.orders.createOrder({
-      total_price_cents_usd: 500,
+      total_price_cents_usd: 100,
       project_id: biomass_test_project_id
     });
 
-    expect(parseFloat(createOrderResponse.data.price_cents_usd)).to.be.closeTo(
-      500,
-      1
-    );
-    expect(createOrderResponse.data.patch_fee_cents_usd).to.equal('0.0');
-    expect(createOrderResponse.data.mass_g).to.equal(500000);
```

Comment:
Existing values were not working after we changed billing. This should have been updated, but most importantly caught by the health check. It did, but we weren't paying attention (I don't know where the alerts went). I'll take a closer look when we strengthen the SDK.